Harvest.Web/ClientApp/src/Tickets/TicketListContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+
+import { Ticket } from "../types";
+import { TicketTable } from "./TicketTable";
+
+interface Props {
+  projectId: any;
+}
+interface RouteParams {
```

Comment:
from what i can tell this is never being used/populated.  You seem to be using this from another component and passing props, and if that's the case you should be using that instead of half-using the props.projectId and half using projectId.